testsuite/src/it/java/io/quarkus/ts/startstop/SpecialCharsTest.java

```diff
@@ -183,14 +183,14 @@ public void specialJVM(TestInfo testInfo) throws IOException, InterruptedExcepti
     }
 
     @Test
-    @DisabledOnOs({OS.WINDOWS}) // https://github.com/quarkusio/quarkus/issues/9707
+    @DisabledOnOs(value = {OS.WINDOWS}, disabledReason = "https://github.com/quarkusio/quarkus/issues/42248")
     public void specialDEV(TestInfo testInfo) throws IOException, InterruptedException {
         testRuntime(testInfo, Apps.JAKARTA_REST_MINIMAL, MvnCmds.DEV, ",;~!@#$%^&()");
     }
 
     @Test
     @Tag("native")
-    @DisabledOnOs({OS.WINDOWS}) // https://github.com/quarkusio/quarkus/issues/9707
+    @DisabledOnOs(value = {OS.WINDOWS}, disabledReason = "https://github.com/quarkusio/quarkus/issues/42248")
```

Comment:
As this is native test shouldn't be reason `"Native on Windows is not supported"`. On the next line we disabling the whole test with same issue. I would expect this test to work when the issue is fixed or am I missing something why this should not work on windows?